ROS/osr_control/osr_control/roboclaw_wrapper.py

```diff
@@ -59,7 +59,7 @@ def __init__(self):
                 ('roboclaw_mapping.drive_right_back.channel', None),
                 ('roboclaw_mapping.drive_right_back.ticks_per_rev', None),
                 ('roboclaw_mapping.drive_right_back.gear_ratio', None)
-            ]
+             ]
```

Comment:
```suggestion
            ]
```
this seems to be a typo, if anything this should be 4 spaces back. Now it seems to be 5 spaces in.

Reply:
Good catch! Commit 7bbf40e :)